sqlvm-alwayson-cluster/nested/provisioningNetworkStorageAGSet.json

```diff
@@ -66,7 +66,7 @@
       },
       {  
          "type":"Microsoft.Storage/storageAccounts",
-         "name":"[concat(substring(parameters('sqlStorageAccountName'),0,20), 'fsw')]",
+         "name":"[concat(substring(concat(parameters('sqlStorageAccountName'),uniqueString(parameters('sqlStorageAccountName'))),0,20),'fsw')]",
```

Comment:
You can go one step further and remove the storage account name parameter altogether and just generate it uniquely. See #944